composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "aligent/pinpay",
   "description": "",
   "require": {
-    "php": "~5.5.0|~5.6.0|^7.0.0",
+    "php": "~7.0.0||~7.1.0||~7.2.0||~7.3.0",
```

Comment:
Interesting; I'd never noticed that `||` is the actual documented separator, good to know :+1: